src/start/TutorialScreen.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import boundActions from '../boundActions';
+import { Text, View, Image, StyleSheet } from 'react-native';
+
+import AppIntro from 'react-native-app-intro';
+
+class TutorialScreen extends React.Component {
+
+  onSkipBtnHandle = (index) => {
+    this.props.pushRoute('realm');
+    console.log(index);
+  }
+
+  doneBtnHandle = () => {
+    this.props.pushRoute('realm');
+  }
+  nextBtnHandle = (index) => {
+    console.log(index);
+  }
+  onSlideChangeHandle = (index, total) => {
+    console.log(index, total);
+  }
+  
+  render() {
+    const pageArray = [{
```

Comment:
Extract this outside the render function.
Even better extract it into a separate module.